zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -72,30 +97,79 @@
 
   /**
    * note configurations.
-   *
+   * <p/>
    * - looknfeel - cron
    */
+  @XmlElementWrapper
+  @XmlElement
   private Map<String, Object> config = new HashMap<>();
 
   /**
    * note information.
-   *
+   * <p/>
    * - cron : cron expression validity.
    */
+  @XmlElementWrapper
+  @XmlElement
   private Map<String, Object> info = new HashMap<>();
 
 
-  public Note() {}
+  public Note() {
+  }
 
   public Note(NotebookRepo repo, NoteInterpreterLoader replLoader,
-      JobListenerFactory jlFactory, SearchService noteIndex) {
+              JobListenerFactory jlFactory, SearchService noteIndex) {
     this.repo = repo;
     this.replLoader = replLoader;
     this.jobListenerFactory = jlFactory;
     this.index = noteIndex;
     generateId();
   }
 
+  private static class MyAngularObjectMapList {
+    @XmlAttribute
+    String key;
+    @XmlElementWrapper
+    @XmlElement
+    List<AngularObject> value;
+
+    public MyAngularObjectMapList(String key, List<AngularObject> value) {
+      this.key = key;
+      this.value = value;
+    }
+  }
+
+  private static class AngularObjectMapWrapper {
+    @XmlElementWrapper
+    @XmlElement
+    List<MyAngularObjectMapList> list = new ArrayList<MyAngularObjectMapList>();
+  }
+
+  private static class MapofAngularListsAdapter extends
+      XmlAdapter<AngularObjectMapWrapper, Map<String, List<AngularObject>>> {
+
+    @Override
+    public Map<String, List<AngularObject>> unmarshal(AngularObjectMapWrapper v) throws Exception {
+      HashMap<String, List<AngularObject>> unmarshaledMap =
+          new HashMap<String, List<AngularObject>>();
```

Comment:
Have you heard about java diamond operator? Can it be used here to have shorter code? Or is there some helper methods in JDK to make it even more readable?